examples/query-action/query_action.metatags

```diff
@@ -1,2 +1,2 @@
-description: Example on using Ballerina query action.
-keywords: ballerina by examples, bbe, query, query action, from, join, on, where, let, do, do action, iterable, iterate
+description: Example on performing query actions on iterable items.
+keywords: ballerina examples, query, query action, do action, iterable, iterate
```

Comment:
Edit: keywords: ballerina by example, query, query action, do action, iterable, iterate

Reply:
In all BBE keywords (in other files as well) change "ballerina examples" to "ballerina by example"